plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1057,7 +1057,7 @@ def main():
             smbios=dict(type='str'),
             snapname=dict(type='str'),
             sockets=dict(type='int'),
-            sshkeys=dict(type='str'),
+            sshkeys=dict(type='str', no_log=False),
```

Comment:
This is not very useful for proxmox_kvm as this parameter is actually a public key, and therefore not a secret.

Reply:
Yes, that's why `no_log` is set to `False`, to indicate that it is not secret.

Reply:
Oops, I apparently stopped reading after seeing the parameter `no_log`.

Reply:
:D That does happen ;-) Thanks for checking!